src/client/sound/s_main.c

```diff
@@ -77,7 +139,8 @@ void S_Frame(void) {
 	}
 
 	if (s_reverse->modified) {
-		Mix_SetReverseStereo(MIX_CHANNEL_POST, s_reverse->integer);
+		// TODO
+		//Mix_SetReverseStereo(MIX_CHANNEL_POST, s_reverse->integer);
```

Comment:
Honestly, this is a stupid option. I don't know anyone who needs it. We can remove this if it's going to take more than 5 seconds to fix.